trim.production.js

```diff
@@ -1,10 +1,16 @@
-var request = require('request');
-var assert = require('assert');
-var debug = require('debug')('trim');
-var R = require('ramda-extended');
-var url;
-var token;
+var request = require('request')
+var assert = require('assert')
+var debug = require('debug')('trim')
+var R = require('ramda-extended')
+var url
+var token
 
+var PRIVACY_LEVELS = {
+  PUBLIC: 1,
+  PRIVATE: 2
+}
+
+var isValidPrivacyLevel = R.contains(R.__, R.values(PRIVACY_LEVELS))
```

Comment:
After looking up R.__ I understand whats going on here and that trick would have come in useful for me recently.